tests/ci/docker_images/linux-aarch/ubuntu-20.04_clang-7x-bm-framework/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu-20.04-aarch:clang-7x
+
+SHELL ["/bin/bash", "-c"]
+
+RUN set -ex && \
+    apt-get update && \
+    apt-get -y --no-install-recommends upgrade && \
+    apt-get -y --no-install-recommends install \
+    python3.8 \
+    python3-pip \
+    make && \
+    pip install pandas && \
+    pip install numpy && \
+    pip install boto3 && \
```

Comment:
Nit: the dependency management should be moved to the `tests/ci/benchmark_framework` and have the version locked.

Reply:
Locking the version of the dependencies makes sense, I will do that in my next push. What do you mean by moving dependency management to `tests/ci/benchmark_framework` though? I thought doing it in the dockerfile would make the most sense.

Reply:
>  I thought doing it in the dockerfile would make the most sense.

How to manage/lock python dependencies through container? Refer CDK or other online example on python dependency management.

This comment is not required for merging this PR.